pyproject.toml

```diff
@@ -21,12 +21,12 @@ eq-translations = {git = "https://github.com/ONSDigital/eq-translations.git"}
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.5"
 flake8 = "^4.0.1"
-pylint = "^2.12.2"
+pylint = "^2.15.3"
```

Comment:
There is a later version of pylint so maybe if you pin to that it will solve your linting issue (`"^2.15.4"`).

Reply:
Cyclic import still occurs. I cannot create the error locally, only shows up during PR